icuSources/common/msvcres.h

```diff
@@ -11,7 +11,7 @@
 //
 
 /*
-These are defined before _foundation_unicode/uversion.h in order to prevent 
+These are defined before unicode/uversion.h in order to prevent 
```

Comment:
It's not important here since it's just a comment, but should we tweak your script to apply another round of /s/unicode/_foundation_unicode after the merge, in case we miss places where this difference really matters?

Reply:
Just testing `unicode/*h` might work... But I'm feeling somewhat nervous by that so I'd rather test `#include unicode/*.h`... IMO the comments are fine since we don't really manually edit the code anyways.